RecoJets/JetProducers/plugins/VirtualJetProducer.cc

```diff
@@ -752,6 +752,8 @@ void VirtualJetProducer::writeJets(edm::Event& iEvent, edm::EventSetup const& iS
         // Here it is assumed that fjJets_ is in decreasing order of pT,
         // which should happen in FastjetJetProducer::runAlgorithm()
         jetArea = M_PI;
+        //rij is properly initialized above
+        [[clang::suppress]]
```

Comment:
`rij` is properly initialized above. It is false positive report from cland-analyzer.